README.md

```diff
@@ -76,16 +76,35 @@ Now you can visit the site at http://0.0.0.0:8000/nyc2019/.
   way that I know of to make user group changes take effect).
 * Clone this repo and change to its directory.
 * Create a virtualenv (using Python 2.7, sorry)
+
+    ```$ mkvirtualenv opendebates -p `which python2.7` ```
+
 * Activate virtualenv
-* ```pip install -r requirements/dev.txt```
-* ```fab up``` (smoketest may fail)
-* ```fab migrate```
-* ```fab smoketest``` (should pass now)
-* ```fab createsuperuser``` (answer prompts to create superuser)
+
+    ```$ workon opendebates```
+
+* Install the project requirements
+
+    ```pip install -r requirements/dev.txt```
+
+* Build the docker container
+
+    ```docker-compose up --build -d```
+
+* Run migrations in the docker container
+
+    ```docker-compose run web python manage.py migrate --noinput -v 0```
```

Comment:
I thought this didn't work at first because of the `-v 0`. Maybe if it's not harmful we could allow the output?
```suggestion
    ```docker-compose run web python manage.py migrate --noinput```
```